test/jdk/java/awt/ScrollPane/ScrollPaneRemoveAdd.java

```diff
@@ -74,30 +73,31 @@ public void actionPerformed(ActionEvent e) {
             });
             frame.pack();
             frame.setLocationRelativeTo(null);
+            frame.setAlwaysOnTop(true);
```

Comment:
When I run the folder, due to previous test/command prompt the frame used to show up behind command prompt, so to be on safer side I used always on top. When we run individual test, it works fine.

Reply:
It may still not work… You have to disable keyboard input timeout by setting `ForegroundLockTimeout` to zero as explained in [requirements for Windows](https://wiki.openjdk.org/display/ClientLibs/Automated+client+GUI+testing+system+set+up+requirements#AutomatedclientGUItestingsystemsetuprequirements-Windows-SpecificSystemsetupnotes).

Most likely, you don't want to do it on the system that you regularly use, otherwise apps could steal focus.

Reply:
Its working fine after setting to always on top, tested multiple times.